test/api/requests/devfile.invalid

```diff
@@ -8,10 +8,10 @@ projects:
 tools:
   - name: theia-ide
```

Comment:
@metlos isn't this name shouldn't be here anymore?

Reply:
Good catch @garagatyi

Reply:
@garagatyi since you have spotted this can you please remove it?

Reply:
sure

Reply:
done